explanations/signing/Apple_signing_and_notarization.rst

```diff
@@ -0,0 +1,122 @@
+Signing and Notarization
+========================
+
+Certificates
+------------
+
+For certificate rotation, see the signing page in `RelEng Confluence space <https://mozilla-hub.atlassian.net/wiki/spaces/RelEng/overview>`__.
```

Comment:
Please update that wiki page to talk about how to rotate provisioning profiles at well, since it is now a required part of certificate rotation.